tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_archlinux_inventory.yaml

```diff
@@ -34,22 +34,4 @@
         - path:
             value: NVD_JSON_PATH
         - update_interval:
-            value: '5s'
-  - section: sca
-    elements:
-    - enabled:
-        value: 'no'
-  - section: rootcheck
-    elements:
-    - disabled:
-        value: 'yes'
-  - section: syscheck
-    elements:
-    - disabled:
-        value: 'yes'
-  - section: wodle
-    attributes:
-      - name: 'syscollector'
-    elements:
-      - disabled:
-          value: 'yes'
\ No newline at end of file
+            value: '5s'
```

Comment:
Maybe we should keep these modules turned off, to improve the stability of the test